tasks/web-scraping/src/main.rs

```diff
@@ -0,0 +1,53 @@
+use regex::Regex;
+use std::error::Error;
+/// Create a program that downloads the time from this URL: http://tycho.usno.navy.mil/cgi-bin/timer.pl
+/// and then prints the current UTC time by extracting just the UTC time from the web page's HTML.
+/// For an introduction to the ecosystem of web scraping in Rust,
+/// read https://kadekillary.work/post/webscraping-rust/
+
+const A_URL: &str = "http://tycho.usno.navy.mil/cgi-bin/timer.pl";
+fn extract_timestamp(raw_html: &str) -> Option<regex::Match> {
+    // unicode below is points for < and >
+    let pattern = Regex::new(r"\u003CBR\u003E(?P<time>.*) UTC").unwrap();
```

Comment:
Why use unicode escapes here? Can't you just use `<BR>` instead?

Reply:
I had confused myself seeing compiler errors and `None` results when initially trying to escape them. So, fresh day and the edit clears it up :)